bundle/terranova/tnresources/job.go

```diff
@@ -24,6 +24,10 @@ func (*ResourceJob) PrepareState(input *resources.Job) *jobs.JobSettings {
 	return &input.JobSettings
 }
 
+func (*ResourceJob) RemapState(jobs *jobs.Job) *jobs.JobSettings {
```

Comment:
It could be worth adding type aliases for readability. We have 3 types that we're dealing with here:
* Config type: `resources.Job`
* State type: `jobs.JobSettings`
* Remote type: `jobs.Job`

If we alias each of these to e.g. `JobConfig`, `JobState`, `JobRemote`, then it is unambiguous in which direction the remapping happens. It's not clear without context that this function maps from the remote type to the state type.